controllers/apply/awards-for-all/fields.js

```diff
@@ -1852,14 +1998,21 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             }),
             type: 'text',
             isRequired: true,
-            schema: Joi.string().required(),
+            schema: Joi.string().max(FREE_TEXT_MAXLENGTH.large).required(),
             messages: [
                 {
                     type: 'base',
                     message: localise({
                         en: 'Enter the full legal name of the organisation',
                         cy: ''
                     })
+                },
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `Full legal name of the organisation must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
'Full legal name of organisation must be ${FREE_TEXT_MAXLENGTH.large} characters or less`